src/plugins/user-activity-events/client.js

```diff
@@ -23,12 +23,17 @@ function throttle(fn) {
 }
 
 module.exports = function recordUserEvents(client) {
+
 	if (document.addEventListener) {
-		document.addEventListener('click', throttle(function() {
-			client.sendEvent('userIsActive');
-		}));
-		document.addEventListener('keydown', throttle(function() {
+		var listener = throttle(function userActivityListener() {

```

Comment:
Ok, thanks. I'm good leaving the function names present. then. If we're satisfied, can you merge and version ifrau @dlockhart? Thanks :-)